### PR TITLE
Bugfix/player 3289  Language menu is closed by clicking on Language icon after switching to window mode 

### PR DESCRIPTION
**Pull request by KseniiaSokolovaEpam**
Bug fix for
[https://jira.corp.ooyala.com:8443/browse/PLAYER-3295] and
[https://jira.corp.ooyala.com/browse/PLAYER-3289]
When we click on the fullscreen icon in fullscreen mode popovers are closing;
When we change a language in a multiaudio popover (large screen), the popover is closing;
It was checked it in the test.

\+ path to const was fixed (for quality popover)
\+ some changes for eslint

### CODE REVIEW

File: js/skin.js [hunk 1]
```diff
@@ -155,6 +160,7 @@ var Skin = React.createClass({
   /**
    * @public
    * @description set isVrMouseMove to false for click event
+   * @return {undefined}
```
Comment (pilievOoyala) [inline js/skin.js]: We should take this out since it doesn't return anything.

File: js/skin.js [hunk 2]
```diff
@@ -165,17 +171,19 @@ var Skin = React.createClass({
   /**
    * @public
    * @description set accessibilityControlsEnabled to true for focus event
+   * @return {undefined}
```
Comment (pilievOoyala) [inline js/skin.js]: We should take this out since it doesn't return anything.

**Comment by pilievOoyala** [comment]
LGTM. Except for the two minor jsdoc things.

File: js/skin.js [hunk 3]
```diff
@@ -122,6 +125,7 @@ var Skin = React.createClass({
   /**
    * @public
    * @description the function is called when we stop the rotation
+   * @return {undefined}
```
Comment (aeng7) [inline js/skin.js]: Another undefined jsdoc return

**Comment by aeng7** [comment]
LGTM other than Peter's comments

File: js/skin.js (hunk 1 above)
Comment (KseniiaSokolovaEpam) [reply js/skin.js]: \+

File: js/skin.js (hunk 2 above)
Comment (KseniiaSokolovaEpam) [reply js/skin.js]: \+

File: js/skin.js (hunk 3 above)
Comment (KseniiaSokolovaEpam) [reply js/skin.js]: \+

**Pull request merged**